src/routes/safe/components/Balances/SendModal/screens/SendFunds/TokenSelectField/style.ts

```diff
@@ -5,8 +5,8 @@ import { sm } from 'src/theme/variables'
 export const useSelectedTokenStyles = makeStyles(
   createStyles({
     container: {
-      minHeight: '55px',
-      padding: 0,
+      background: 'none !important',
```

Comment:
You could potentially move this into overrides too.

Reply:
This targets the `MenuItem` element when the Token select is closed. I wasn't able to find an overwrite path within the global overwrites to only target `MenuItem` in this state but not the dropdown state.